galaxy/wrapper/bamCompare.xml

```diff
@@ -187,25 +187,30 @@
             <param name="outFileFormat" value="bedgraph"/>
             <param name="binSize" value="5"/>
             <param name="type" value="ratio"/>
-            <output name="outFileName" file="bamCompare_result1.bg" ftype="bedgraph"/>
+            <output name="outFileName" ftype="bedgraph">
+                <assert_contents>
+                    <has_text_matching expression="chrM\t0\t16569\t1"/>
```

Comment:
why did you choose to drop the file comparison? 

Its a pretty good test to see the differences between both versions.

Reply:
There is only one line in the output :)

Reply:
then please add an assert for the line_counts as well :)